rpcs3/Gui/LLEModulesManager.cpp

```diff
@@ -8,8 +8,9 @@
 #include "Emu/state.h"
 #include "Emu/FS/VFS.h"
 
-LLEModulesManagerFrame::LLEModulesManagerFrame(wxWindow* parent) : wxFrame(parent, wxID_ANY, "LLEModulesManagerFrame")
+LLEModulesManagerFrame::LLEModulesManagerFrame(wxWindow* parent) : wxFrame(parent, wxID_ANY, "LLE modules manager frame")
```

Comment:
Would be nice to remove the "frame" at the end of the name. There's no need to indicate to an user, that it's a window frame.